### PR TITLE
setup sdk

### PR DESCRIPTION
**Pull request by haybankz**
### Description
sets up pylons_fultter sdk

### CODE REVIEW

File: lib/screens/home_screen.dart [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                ),
+              ),
+            ),
+            const VerticalSpace(6),
+            Expanded(
+              child: PageView(
+                // allowImplicitScrolling: false,
+                controller: _pageController,
+                physics: const NeverScrollableScrollPhysics(),
+                onPageChanged: (int page) {
```
Comment (kjawadDeveloper2) [inline lib/screens/home_screen.dart:114]: Dont use setState.

File: lib/screens/home_screen.dart [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                width: screenSize.width(percent: 60),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.only(left: 10.0),
+                      child: IconButton(
+                          onPressed: () {
+                            setState(() {
+                              _currentPage = _currentPage > 0 ? _currentPage - 1 : 0;
```
Comment (kjawadDeveloper2) [inline lib/screens/home_screen.dart]: No set state

File: lib/screens/home_screen.dart [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  Widget _buildStepTitle(int index) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        Text(
+          title[index],
+          style: Theme.of(context)
+              .textTheme
+              .bodyText2!
+              .copyWith(color: _currentPage == index ? Colors.black : Colors.grey),
```
Comment (kjawadDeveloper2) [inline lib/screens/home_screen.dart]: Colors should not be define outside app theme.

File: lib/screens/mint_screen.dart [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+          children: [
+            const _ImageWidget(imageUrl: kImage),
+            const SizedBox(height: 10),
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 20),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Text("Mona Lisa", style: Theme.of(context).textTheme.headline5!.copyWith(
+                    color: Colors.black, fontWeight: FontWeight.w600,
```
Comment (kjawadDeveloper2) [inline lib/screens/mint_screen.dart]: Dont define color outside Apptheme.

File: lib/screens/upload_screen.dart [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+              ),
+              const SizedBox(height: 20)
+            ]
+          ),
+          Positioned(
+            child: Visibility(
+              visible: showError,
+              child: _ErrorMessageWidget(
+                onClose: (){
+                  setState(() {
```
Comment (kjawadDeveloper2) [inline lib/screens/upload_screen.dart:49]: No set state.

File: lib/screens/upload_screen.dart [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+            )
+          ),
+          child: Image.asset("assets/icons/file.png",),
+        ),
+        const VerticalSpace(5),
+        Text("SVG file", style: Theme.of(context).textTheme.subtitle2!.copyWith(
+          color: Colors.grey
+        ),),
+        Text("40MB limit", style: Theme.of(context).textTheme.subtitle2!.copyWith(
+            color: Colors.grey
```
Comment (kjawadDeveloper2) [inline lib/screens/upload_screen.dart]: No colors.

File: lib/widgets/pylons_round_button.dart [hunk 7]
```diff
@@ -13,7 +13,7 @@ class PylonsRoundButton extends StatelessWidget {
       child: const Icon(Icons.arrow_forward_ios, color: Colors.white),
```
Comment (kjawadDeveloper2) [inline lib/widgets/pylons_round_button.dart]: Remove color from here. Add it in App theme.

File: lib/main.dart [hunk 8]
```diff
@@ -1,7 +1,12 @@
 import 'package:easel_flutter/screens/routing_screen.dart';
 import 'package:flutter/material.dart';
+import 'package:pylons_flutter/pylons_flutter.dart';
 
 void main() {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  PylonsWallet.setup(mode: PylonsMode.prod, host: 'pylons_easel');
```
Comment (kjawadDeveloper2) [inline lib/main.dart]: change host to ease.

File: android/app/src/main/AndroidManifest.xml [hunk 9]
```diff
@@ -31,6 +34,17 @@
...
+
+            <!-- Pylons Links -->
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <!-- Accepts URIs that begin with YOUR_SCHEME://YOUR_HOST -->
+                <data
+                    android:scheme="pylons"
+                    android:host="pylons_easel" />
```
Comment (kjawadDeveloper2) [inline android/app/src/main/AndroidManifest.xml]: only write easel in host.

File: ios/Runner/Info.plist [hunk 10]
```diff
@@ -41,5 +41,23 @@
...
+	<key>CFBundleURLTypes</key>
+    <array>
+        <dict>
+            <key>CFBundleTypeRole</key>
+            <string>Editor</string>
+            <key>CFBundleURLName</key>
+            <string>tech.pylons.easel</string>
+            <key>CFBundleURLSchemes</key>
+            <array>
+                <string>pylons-pylons_easel</string>
```
Comment (kjawadDeveloper2) [inline ios/Runner/Info.plist]: Only write easel. Ios app will not work if the schema contains _

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]
Kindly solve these issues. Also delete the .idea file from the code. It shouldn't be uploaded.

File: ios/Runner/Info.plist (hunk 10 above)
Comment (haybankz) [reply ios/Runner/Info.plist]: I'll suggest that this should be stated in the SDK documentation

**Comment by haybankz** [comment]
@kjawadDeveloper2  request changes have been implemented

File: ios/Runner/Info.plist (hunk 10 above)
Comment (kjawadDeveloper2) [reply ios/Runner/Info.plist]: Yes you are right and i will add it today. Thanks for mentioning

File: pubspec.yaml [hunk 11]
```diff
@@ -85,18 +85,17 @@ flutter:
...
-  #       - asset: fonts/Schyler-Regular.ttf
-  #       - asset: fonts/Schyler-Italic.ttf
-  #         style: italic
-  #   - family: Trajan Pro
-  #     fonts:
-  #       - asset: fonts/TrajanPro.ttf
-  #       - asset: fonts/TrajanPro_Bold.ttf
-  #         weight: 700
-  #
+  fonts:
```
Comment (kjawadDeveloper2) [inline pubspec.yaml]: Since Inter is a google font . So using google fonts package we can save this space that ttf occupies.

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]

**Comment by haybankz** [comment]
@kjawadDeveloper2 changes have been implemented

File: pubspec.yaml [hunk 12]
```diff
@@ -85,17 +87,17 @@ flutter:
   # in this "flutter" section. Each entry in this list should have a
   # "family" key with the font family name, and a "fonts" key with a
   # list giving the asset and other descriptors for the font. For
-  fonts:
```
Comment (kjawadDeveloper2) [inline pubspec.yaml]: Did you also removed these files from assets folder?

Comment (haybankz) [reply pubspec.yaml]: yes, i did

**Review by kjawadDeveloper2 — APPROVED** [review]
Looks Good from my side.

**Review by starhelios — APPROVED** [review]

**Pull request merged**